Updates to .NET contributing guide (#410)
MicrosoftDocs/Contribute: Contribute/dotnet/dotnet-pr-review.md

- **Small PRs** - Small PRs are single bug fix: typos, broken links, small code changes, or similar changes.
- **Major contributions** - Major contributions are significant edits to an existing article, new articles, or edits to a number of articles.

> - **Work in progress** - Authors can request an in-progress review by opening a *draft* pull request.

Reviewer: "Draft" status typically means "not ready for review".

```suggestion
- **Work in progress** - Authors can can create a PR that is marked as not ready for review yet by opening a *draft* pull request.
```

- The article meets [style and grammar guidelines](dotnet-style-guide.md) and follows our [voice and tone principles](dotnet-voice-tone.md).
- All links should resolve correctly.

Content team members will review the PR and submit the review with comments. If only minor changes are requested, team members may "approve" the PR with the feedback. The author can then address the feedback and merge the PR. Most reviews will request changes and when those changes are made, the reviewer will review again.

If the edits require a technical review, the content team reviewer will request a review from the appropriate product team member.

### Review draft pull requests

> New authors may want feedback earlier in the process. They can open a draft PR and request an early review. These early reviews focus on the structure of the article: the outline, the overall content, and the samples. These reviews do not include a thorough check for grammar and correct links.

Reviewer: ```suggestion
You may want feedback earlier in the process. Open a draft PR and add a comment that requests an early review. These early reviews focus on the structure of the article: the outline, the overall content, and the samples. These reviews do not include a thorough check for grammar and correct links.
```
